testsuite/libmapiproxy/openchangedb_logger.c

```diff
@@ -41,10 +44,26 @@
 } while(0)
 
 
+struct openchangedb_context_checker {
+        int get_SpecialFolderID;
+        int get_SystemFolderID;
+        int get_PublicFolderID;
+        int get_distinguishedName;
+        int get_MailboxGuid;
+        int get_MailboxReplica;
+        int get_PublicFolderReplica;
+        int get_parent_fid;
+        int get_MAPIStoreURIs;
+        int get_mapistoreURI;
+        int set_mapistoreURI;
+        int get_fid;
+        int get_ReceiveFolder;
+};
+
+
 static TALLOC_CTX *mem_ctx;
 static struct openchangedb_context *oc_ctx;
-
-static struct openchangedb_context functions_called;
+static struct openchangedb_context_checker functions_called;
 
```

Comment:
What is the benefit of introducing this openchangedb_context_checker instead of keeping using openchangedb_context for functions_called?